packages/serialization/json/src/jsonParseNode.ts

```diff
@@ -53,7 +59,11 @@ export class JsonParseNode implements ParseNode {
 		}
 		return undefined;
 	}
-	public getCollectionOfObjectValues = <T extends Parsable>(method: ParsableFactory<T>): T[] | undefined => {
+	public getCollectionOfObjectValues = <T extends Parsable>(method: ParsableFactory<T>): T[] | null | undefined => {
+		if (this._jsonNode === null) {
+			return null;
+		}
+
```

Comment:
what about these cases?
should I also remove `null` handling?
Will they handle it properly?